bridge-impl/src/main/java/com/liferay/faces/bridge/internal/BridgeURLBase.java

```diff
@@ -409,9 +407,7 @@ protected ResourceURL createResourceURL(FacesContext facesContext, boolean modeC
 				// Only add the "javax.portlet.faces.PortletMode" parameter if it has a valid value.
 				if (firstParameterValue != null) {
 
-					PortletModeValidatorFactory portletModeValidatorFactory = (PortletModeValidatorFactory)
-						BridgeFactoryFinder.getFactory(PortletModeValidatorFactory.class);
-					PortletModeValidator portletModeValidator = portletModeValidatorFactory.getPortletModeValidator();
+					PortletModeValidator portletModeValidator = PortletModeValidatorFactory.getPortletModeValidatorInstance();
```

Comment:
@ngriffin7a I did not format this file because it hadn't been formatted (so the formatting changes were massive). These lines are over 120 chars.